ome_zarr/format.py

```diff
@@ -215,5 +241,60 @@ def validate_well_dict(
         if well["columnIndex"] != columns.index(column):
             raise ValueError(f"Mismatching column index for {well}")
 
+    def generate_coordinate_transformations(
+        self, shapes: List[tuple]
+    ) -> Optional[List[List[Dict[str, Any]]]]:
+
+        data_shape = shapes[0]
+        coordinate_transformations: List[List[Dict[str, Any]]] = []
+        # calculate minimal 'scale' transform based on pyramid dims
+        for shape in shapes:
+            assert len(shape) == len(data_shape)
+            scale = [full / level for full, level in zip(data_shape, shape)]
+            coordinate_transformations.append([{"type": "scale", "scale": scale}])
+
+        return coordinate_transformations
+
+    def validate_coordinate_transformations(
+        self,
+        shapes: List[tuple],
```

Comment:
Feels strange to have this API depending on shapes especially as only the length is used so far. Given the known assumptions, I wonder if `len(axes)` would be sufficient ?